applications/adv_networking_bench/Dockerfile

```diff
@@ -0,0 +1,38 @@
+# syntax=docker/dockerfile:1
+
+# SPDX-FileCopyrightText: Copyright (c) 2024 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+ARG BASE_IMAGE
+ARG GPU_TYPE
+
+FROM ${BASE_IMAGE} as base
+
+ARG UBUNTU_VERSION=22.04
+ARG OFED_VERSION=24.01-0.3.3.1
```

Comment:
Is this version standard? Seems that 24.07-0.6.1.0 will be best for stack alignment, unless there is a clear reason to pin to 24.01.

Reply:
There's no standard version of OFED, but the ANO and Holoscan both require it. we should pick one version to use for both.

Reply:
@tbirdso @cliffburdick OK to use 24.07-0.6.1.0?

Reply:
yes